src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -0,0 +1,1189 @@
+      {(() => {
+        switch (true) {
+          case module.contentList.length < 5:
+            return (
+              <div className={'grid grid-cols-2'}>
+                <ul className="grid gap-4">{children}</ul>
+                <p className="mb-6 prose text-gray-900 dark:prose-dark md:prose-lg md:dark:prose-lg-dark dark:text-gray-100 dark:prose-a:text-blue-300 dark:hover:prose-a:text-blue-200 prose-a:text-blue-500 hover:prose-a-:text-blue-600 ">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+                  irure dolor in reprehenderit in voluptate velit esse cillum
+                  dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+                  cupidatat non proident, sunt in culpa qui officia deserunt
+                  mollit anim id est laborum.
+                </p>
+              </div>
+            )
+          case module.contentList.length > 5 && module.contentList.length <= 10:
+            return (
+              <div>
+                <p className="mb-6 prose text-gray-900 dark:prose-dark md:prose-lg md:dark:prose-lg-dark dark:text-gray-100 dark:prose-a:text-blue-300 dark:hover:prose-a:text-blue-200 prose-a:text-blue-500 hover:prose-a-:text-blue-600 ">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+                  irure dolor in reprehenderit in voluptate velit esse cillum
+                  dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+                  cupidatat non proident, sunt in culpa qui officia deserunt
+                  mollit anim id est laborum.
+                </p>
+                <ul className="grid grid-rows-5 grid-flow-col gap-4">
+                  {children}
+                </ul>
+              </div>
+            )
+          default:
+            return <ul className="grid grid-cols-1">{children}</ul>
+        }
+      })()}
+    </div>
+  )
+}
+
+const CollectionContent = (props: any) => {
+  return props.contentList.map((content: any, index: number) => {
+    switch (content.type) {
+      case 'module':
+        return (
+          <ModuleCollection module={content}>
+            <CollectionContent {...props} contentList={content.contentList} />
+          </ModuleCollection>
+        )
+      case 'lesson':
+        return (
+          <LessonLinkResource
+            lesson={content}
+            index={index}
+            completedLessonSlugs={props.completedLessonSlugs}
+          />
+        )
```

Comment:
In the case of the content type being lesson, we return the `LessonLinkResource`. There is no need to recurse here since this isn't a collection and won't have children.